app/controllers/admin/api/services_controller.rb

```diff
@@ -46,7 +46,9 @@ def index
   ##~ op.parameters.add :name => " ", :dataType => "custom", :paramType => "query", :allowMultiple => true, :description => "Extra parameters"
   #
   def create
-    service = current_account.create_service(service_params)
+    service = current_account.accessible_services.new(service_params)
+    create_service = CreateServiceWithBackendApi.new(service: service, backend_api_id: params[:backend_api_id].presence)
```

Comment:
If `backend_api_id` is an optional parameter I will call it `CreateServiceProcedure` (because CreateServiceService is ugly)